tutorials/large-messages/index.mdx

```diff
@@ -0,0 +1,377 @@
+---
+meta:
+  title: Create a serverless architecture that manage large messages, with Scaleway Messaging and Queuing NATS, Serverless Functions and Object Storage.
```

Comment:
```suggestion
  title: Set up a serverless architecture for handling large messages using Scaleway's NATS, Serverless Functions, and Object Storage.
```
@RoRoJ Don't know if this makes sense?

Reply:
I think we still need Messaging & Queuing in there as the product name... how about

Set up a serverless architecture for handling large messages using Scaleway's Messaging & Queuing: NATS, Serverless Functions, and Object Storage.